### DISCUSSION TITLE
Query Error: Worker call timed out!

### DISCUSSION THREAD

**Discussion by kerbersystems** [Budibase/budibase#5596] [Help]
I'm working with a large database and when trying to create a query, it times out. What can I do to try and fix this?

**Comment by shogunpurple** [comment]
@kerbersystems are you running in budibase cloud?

Please refer to this discussion: https://github.com/Budibase/budibase/discussions/5139

**Reply by kerbersystems** [reply]
@shogunpurple I am running on-prem

**Reply by shogunpurple** [reply]
Ok cool - in that case you can make the update described in this comment to increase your timeout.
https://github.com/Budibase/budibase/discussions/5139#discussioncomment-2455442
